### DISCUSSION TITLE
Is it possible to schedule 1 notification by the days of the week?

### DISCUSSION THREAD

**Discussion by ckoziel** [thudugala/Plugin.LocalNotification#464] [Q&A] [answered]
It is possible to schedule notifications by days of the week. For example, I would like a notification to run every Monday, Tuesday, and Friday. It is possible to setup 1 notification on multiple days of the week that is recurring. Something like this?

                    Schedule =
                            {
                                NotifyTime = _notification.StrTime,
                                RepeatType = NotificationRepeat.Daily,
                                NotifyRepeatInterval = {DayOfWeek.Monday, DayOfWeek.Tuesday, DayOfWeek.Friday },

                            },

**Comment by thudugala** [comment, marked as answer]
@ckoziel No, you cannot. limitation from the OS itself.
